docs/modal.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Modal Component</title>
+    <link rel="stylesheet" href="/styles/main-style.css">
+    <link rel="stylesheet"
+    href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css">
```

Comment:
It's better to define stylesheets before the title.

Reply:
I did not took this in consideration. Thanks for pointing it out, I defined stylesheets before the title.

 PS: This is the preview of the modal component- https://deploy-preview-14--parts-builder.netlify.app/docs/modal